testsuite/features/support/commonlib.rb

```diff
@@ -51,7 +51,7 @@ def product_version
 
 def use_salt_bundle
   # Use venv-salt-minion in Uyuni, or SUMA Head, 4.2 and 4.3
```

Comment:
Pretty irrelevant nitpick, but I guess the comment should mention 5.0 too if it stays there ---> " ...4.2, 4.3 and 5.0"

Reply:
thanks. I just fixed that and did a rebase.

Reply:
We need a refactor wherever we use this method.
This method is not useful anymore, as we will use salt bundle by default in all our released versions as per 2024.